snippets/reduceWhich.md

```diff
@@ -0,0 +1,25 @@
+### reduceWhich
+
+By setting some rules, get Minimum or Maximum value in array .
```

Comment:
The description should explain what the snippet method is used for.

Reply:
Fixed